completion/available/sqlmap.completion.bash

```diff
@@ -20,7 +20,7 @@ function _sqlmap() {
 
 		# List directory content
 		--tamper)
-			IFS=' ' read -r -a COMPREPLY <<< "$(compgen -W "${tamper:-}" -- "$cur")"
+			mapfile -t COMPREPLY <<< "$(compgen -W "${tamper:-}" -- "$cur")"
```

Comment:
According to README, Bash-it supports Bash >= 3.2, though mapfile is only available since Bash 4.0.

Reply:
excellent point. I'll look for an acceptable backward compatible solution (although I must admit, dumping bash<4.4 is tempting)

Reply:
I think Bash-3.2 support is kept for macOS.

Reply:
> I'll look for an acceptable backward compatible solution

One way is, as I implied in https://github.com/Bash-it/bash-it/pull/2282#discussion_r1929782708, to use `IFS=$'\n' read -r -d '' -a COMPREPLY <<< "$(compgen ...)"`. However, in Bash >= 4.0, the current version `mapfile -t COMPREPLY <<< "$(compgen ...)"` would be faster. In Bash >= 5.3, the most robust way is to use `compgen -V COMPREPLY ...`. You can prepare a shell function (whose implementation depends on the Bash version) and call that function everywhere you want to obtain the list from `compgen`.

Reply:
Nahh, I am just going to roll them all back to bash 3 code for now. that is the easiest. if bash is stuck at 3.2 on macos till this very day, it won't change anytime soon.